browser-test/src/admin_program_creation.test.ts

```diff
@@ -181,6 +181,18 @@ describe('program creation', () => {
     ])
   })
 
+  it('delete first block and edit', async () => {
```

Comment:
Might be worth adding a test to delete the last block too :D

Reply:
Added test that deletes the last block